packages/components/src/TitleBlockZen/subcomponents/NavigationTabs.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import classnames from "classnames"
+import { TitleBlockVariant } from "../types"
+import { NON_REVERSED_VARIANTS } from "../utils"
+import styles from "./NavigationTabs.module.scss"
+
+export type CustomNavigationTabProps = Omit<NavigationTabProps, "render"> & {
+  className: string
```

Comment:
Oh no. Erm...should we flag it as deprecated? 😅 

Edit: Wait! It's used in `render` and is completely unusable by the consumers. So it's useless? Bah I guess just leave it. It doesn't make any sense anymore.